src/options.ts

```diff
@@ -125,12 +115,47 @@ export function createMatcher(options: RequiredOptions, parser: string, defaults
   }
 
   return {
-    hasStaticAttr: (name: string) => hasMatch(name, staticAttrs, staticAttrsRegex),
-    hasDynamicAttr: (name: string) => hasMatch(name, dynamicAttrs, dynamicAttrsRegex),
+    hasStaticAttr: (name: string) => {
+      // If the name looks like a dynamic attribute we're not a static attr
+      // Only applies to Vue and Angular
+      let newName = nameFromDynamicAttr(name, parser)
+      if (newName) return false
+
+      return hasMatch(name, staticAttrs, staticAttrsRegex)
+    },
+
+    hasDynamicAttr: (name: string) => {
+      // This is definitely a dynamic attribute
+      if (hasMatch(name, dynamicAttrs, [])) return true
+
+      // If the name looks like a dynamic attribute compare the actual name
+      // Only applies to Vue and Angular
+      let newName = nameFromDynamicAttr(name, parser)
+      if (!newName) return false
+
+      return hasMatch(newName, staticAttrs, staticAttrsRegex)
+    },
+
     hasFunction: (name: string) => hasMatch(name, functions, functionsRegex),
   }
 }
 
+function nameFromDynamicAttr(name: string, parser: string) {
+  if (parser === 'vue') {
+    if (name.startsWith(':')) return name.slice(1)
+    if (name.startsWith('v-bind:')) return name.slice(7)
+    if (name.startsWith('v-')) return name
```

Comment:
Do we want to slice until the `=` or `:`?

Reply:
Nope, the parser has already split the name and value and this only matches against the name.

As for the `:` I think that is only useful with v-bind. Can always change this in the future if we need to.